front_end/html/exercise.html

```diff
@@ -198,10 +198,19 @@
                     data_button.innerHTML = "Show";
                 }
             }
+            function showUsers(selection) {
+                var partner_selection = document.getElementById("partner_selection");
+                if (selection == "Yes") {
+                    partner_selection.style.display = "inline";
+                }
+                else {
+                    partner_selection.style.display = "none";
+                }
+            }
         </script>
 
-        <!--The next four divs contain all of the modals for this page-->
-          
+        <!--The next five divs contain all of the modals for this page-->
```

Comment:
It's probably not important to indicate the number of divs in the comment. If you remove that, then the comment won't need to be updated next time something changes.